src/events/messageCreate.ts

```diff
@@ -56,11 +56,12 @@ const MessageEvent: SlashLevel.IEvent = {
 				guildID: message.guild.id,
 				userID: message.author.id,
 				xp: 0,
+				messages: 0,
```

Comment:
again, why do we need to track message count?

Reply:
explained 

> just for storing more statistics. I'm currently working more features for statistics. Next time will need this field for calculating level.